pkg/cmd/cobra/config.go

```diff
@@ -295,50 +290,51 @@ type LogAggregateConfig struct {
 }
 
 type LogFilterConfig struct {
-	LibBPF bool                `mapstructure:"libbpf"`
-	In     LogFilterAttributes `mapstructure:"in"`
-	Out    LogFilterAttributes `mapstructure:"out"`
+	Include LogFilterAttributes `mapstructure:"include"`
+	Exclude LogFilterAttributes `mapstructure:"exclude"`
 }
 
 type LogFilterAttributes struct {
-	Msg   []string `mapstructure:"msg"`
-	Pkg   []string `mapstructure:"pkg"`
-	File  []string `mapstructure:"file"`
-	Level []string `mapstructure:"lvl"`
-	Regex []string `mapstructure:"regex"`
+	Msg    []string `mapstructure:"msg"`
+	Pkg    []string `mapstructure:"pkg"`
+	File   []string `mapstructure:"file"`
+	Level  []string `mapstructure:"lvl"`
+	Regex  []string `mapstructure:"regex"`
+	LibBPF []bool   `mapstructure:"libbpf"`
 }
 
-func getLogFilterAttrFlags(filterOut bool, attrs LogFilterAttributes) []string {
+func getLogFilterAttrFlags(suffix string, attrs LogFilterAttributes) []string {
```

Comment:
```suggestion
func getLogFilterAttrFlags(option string, attrs LogFilterAttributes) []string {
```

In this case `option` might be a better fit.